paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -157,8 +160,9 @@ public class PaparazziPlugin : Plugin<Project> {
         task.compileSdkVersion.set(android.compileSdkVersion())
         task.projectResourceDirs.set(
           project.provider {
-            val generateResValuesDirs = project.tasks.withType(GenerateResValues::class.java).filter { it.variantName == variant.name }.map { it.resOutputDir }
-            localResourceDirs.relativize(projectDirectory) + generateResValuesDirs.map(projectDirectory::relativize)
+            val generateResValuesDirs = project.findTask<GenerateResValues>(variant).map { it.resOutputDir }
+            val extraGeneratedResDirs = project.findTask<MapSourceSetPathsTask>(variant).map { it.extraGeneratedResDir.get().map { File(it) } }.flatten()
```

Comment:
As discussed in https://github.com/cashapp/paparazzi/pull/1307#discussion_r1505398682, this is a temp solution to fetch extra generated res folders. It will be changed to new Gradle API after https://github.com/cashapp/paparazzi/pull/1136 lands